### ISSUE TITLE
Add --verbose option

### ISSUE THREAD

**Issue by roberto-butti** [Hi-Folks/lara-lens#55]
The user can control the output of 
```
php artisan laralens:diagnostic
```
through --show option with values: all|config|runtime|connection|database|migration|php-ext|php-ini|os

add the __--v__ and __--verbose__ and __--all__ options as a shortcut of --show=all.
Something like {--v|verbose|all}

```
php artisan laralens:diagnostic --all
```

**Comment by tweichart** [comment]
fyi just checked for usage of `--v` / `--verbose`, i'd rather suggest to not use this here. it's overwriting default behaviour which imo it shouldn't. also there's no easy way to use these parameters because if used in the signature the command just throws an error (as they're already defined in the framework itself).

therefore i just added the `--a` / `--add` function here ;-)

**Comment by roberto-butti** [comment]
Hi @tweichart , thank you for your PR, it is good for Hacktoberfest 🥳 
I totally agree to use --a or --all to avoid conflicts with verbose option.
I'm going to merge it !!!

**Comment by roberto-butti** [comment]
closed by #57 thank you @tweichart
